wled00/e131.cpp

```diff
@@ -391,9 +390,17 @@ void handleArtnetPollReply(IPAddress ipAddress) {
       break;
   }
 
-  for (uint16_t i = startUniverse; i <= endUniverse; ++i) {
-    sendArtnetPollReply(&artnetPollReply, ipAddress, i);
+  if (DMXMode != DMX_MODE_DISABLED) {
+    for (uint16_t i = startUniverse; i <= endUniverse; ++i) {
+      sendArtnetPollReply(&artnetPollReply, ipAddress, i);
+    }
   }
+
+  #ifdef WLED_ENABLE_DMX
```

Comment:
Why the ifdef?

Reply:
As far as I can tell, this is allowing Art-Net to be translated/proxied to DMX512, like some of the hardware boxes I've been using (Pknight, for example).

This (and the previous Art-Net PR) seems to extend this functionality.

Reply:
The variable `e131ProxyUniverse` used in the ifdef block is only available if `WLED_ENABLE_DMX` is defined ([here](https://github.com/Aircoookie/WLED/blob/main/wled00/wled.h#L414))

Reply:
yeah easier to see from my laptop than looking on my phone. I can see now that you are not moving any existing code into the ifdef, just adding extra which does depend on a value only present based on existing ifdef